python/rmm/tests/test_rmm.py

```diff
@@ -352,3 +352,20 @@ def test_binning_memory_resource(dtype, nelem, alloc, upstream_mr):
     assert rmm.mr.get_current_device_resource_type() is type(mr)
     array_tester(dtype, nelem, alloc)
     rmm.reinitialize()
+
+
+def test_reinitialize_max_pool_size():
```

Comment:
Should there also be a test for a user setting the `initial_pool_size` > `maximum_pool_size`?  I'm assuming that results in `MemoryError`?

Reply:
That's handled by libcudf and it does raise an error. Will add a test for it.

Reply:
Added another test